src/numbers.ts

```diff
@@ -34,7 +46,7 @@ export function kanjiToNumber(text: string): number | null {
 
   while (numbers.length > 1) {
     const [first, second, ...rest] = numbers;
-
+    if (!validNumber(first, second)) return null;
```

Comment:
This is odd, I thought prettier would have automatically converted this to a block if?

Reply:
I’ve already reinstalled prettier at that time. Shall I delete the system-wide package cache and do it again?

Reply:
No it's fine. It turns out prettier doesn't do that after all. I guess I haven't configured it to do that after all.